lyrics-core/src/main/java/com/flipkart/lyrics/specs/AnnotationSpec.java

```diff
@@ -0,0 +1,107 @@
+package com.flipkart.lyrics.specs;
```

Comment:
add a copyright notice like other classes in the repo

Reply:
done